DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -518,10 +518,14 @@ def run_traffic_item(self, device, command, *argv, **kwarg):
             device.applog.info("Clear Stats")
             IxnetworkIxiaClientImpl.ixnet.ClearStats()
         elif command == "clear_traffic":
-            for traffic_item in IxnetworkIxiaClientImpl.tis:
+            traffic_names = kwarg["params"][0]["traffic_names"]
```

Comment:
We can do this by doing below also
```
tis_to_remove = IxnetworkIxiaClientImpl.ixnet.Traffic.TrafficItem.find(Name="|".join(["^%s$" % tname for tname in traffic_names]))
if tis_to_remove:
    tis_to_remove.remove()
```

Reply:
We need to remove traffic items from `IxnetworkIxiaClientImpl.tis` as for it is used during `setup_streams` to generate traffic items, otherwise we will have fail during setup. 
Also intended behavior of function when no `traffic_names` is give, is to delete all traffic items.